src/Features/Core/Portable/UnifiedSuggestions/UnifiedSuggestedActions/UnifiedCodeFixSuggestedAction.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+
+namespace Microsoft.CodeAnalysis.UnifiedSuggestions
+{
+    /// <summary>
+    /// Similar to UnifiedCodeFixSuggestionAction, but in a location that can be used by
+    /// both local Roslyn and LSP.
+    /// </summary>
+    internal class UnifiedCodeFixSuggestedAction : UnifiedSuggestedAction
+    {
+        public CodeFix CodeFix { get; }
+
+        public object Provider { get; }
+
+        public UnifiedSuggestedActionSet? FixAllFlavors { get; }
+
+        public UnifiedCodeFixSuggestedAction(
+            Workspace workspace,
+            CodeFix codeFix,
+            object provider,
+            CodeAction codeAction,
```

Comment:
this ordre is 'interesting'.  given that both workspace and codeAction are passed to the base, i expected them to be the first two parameters.  this is not very important though, just an observation.